RSSReader/View/FeedViewController.m

```diff
@@ -0,0 +1,110 @@
+//
+//  FeedViewController.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 17.11.20.
+//
+
+#import "FeedViewController.h"
+#import "FeedCell.h"
+
+int const kCellHeight = 100;
+
+@interface FeedViewController ()
+
+@end
+
+@implementation FeedViewController
+
+- (void)viewDidLoad {
+    self.navigationController.navigationBar.hidden = YES;
+    [super viewDidLoad];
+    if (!self.feedItemArray) {
+        [self setupFeedItemArray];
+    }
+    [self setupPresenter];
+    [self tableView];
+    [self setupConstraints];
+    [self loadNews];
+}
+
+- (void)setupPresenter {
+    FeedPresenter *presenter = [[FeedPresenter alloc] initWithArray:self.feedItemArray networkManager:[NetworkManager sharedInstance] parser:[RSSParser sharedInstance]];
+    self.presenter = presenter;
+    [presenter release];
```

Comment:
this part shouldn't be within view controller, because the latter must handle UI events mostly.
move presenter initialization in, for example, appdelegate and assign created presenter to this view controller